src/api/app/components/notification_notifiable_link_component.rb

```diff
@@ -69,6 +75,7 @@ def notifiable_link_text
   # rubocop:enable Metrics/CyclomaticComplexity
 
   # rubocop:disable Metrics/CyclomaticComplexity
+  # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
This method is already huge... and I'm adding more, but refactoring this method is out of scope for this PR.